extractors/bookseries/karelians/extractors/kalastus_extractor.py

```diff
@@ -0,0 +1,26 @@
+import re
+
+import core.utils.regex_utils as regexUtils
+from core.pipeline_construction.base_extractor import BaseExtractor
+
+
+class KalastusExtractor(BaseExtractor):
+    extraction_key = 'kalastus'
+
+    def __init__(self, cursor_location_depends_on=None, options=None):
+        super(KalastusExtractor, self).__init__(cursor_location_depends_on, options)
+        self.KALASTUS_PATTERN = r"(?P<kalastus>kalastus)|(kalastamista)|(kalastaa)|(kalastaminen)"

```

Comment:
Code in Finnish. Also, none of these patterns allow for any typos? Did you look into how common mispellings of these words are? I have a recollection that, for example, lower case "L" often gets confused with upper case "i" or 1 by the OCR software.